tests/test_postprocessor_find_info.py

```diff
@@ -0,0 +1,115 @@
+# coding=utf-8
+"""Tests for medusa/test_should_process.py."""
+from medusa.common import Quality
+from medusa.name_parser.parser import NameParser
+from medusa.post_processor import PostProcessor
+
+import pytest
+
+
+class TestPP(PostProcessor):
+    """Test PP class."""
+
+    def __init__(self, file_path, nzb_name):
+        """Initialize the object."""
+        super(TestPP, self).__init__(file_path, nzb_name)
+
+    def _analyze_name(self, name):
+        """Override original name to only parse release name and not match show database."""
+        to_return = (None, None, [], None, None)
+
+        if not name:
+            return to_return
+
+        #  This was changed from original method and removed Try/Expect
+        parse_result = NameParser()._parse_string(name)
```

Comment:
FYI i changed this to not check if show exists in DB. So it only parses the release name